modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceQuickAddToCart.testcase

```diff
@@ -0,0 +1,677 @@
+		}
+
+		task ("When the buyer user opens the minicart") {
+			FrontStore.addInitializerProductToCart(
+				productName = "U-Joint",
+				productQuantity = "1");
+
+			// To be removed when COMMERCE-10472 is fixed
+
+			CommerceNavigator.gotoMiniCart();
+
+			Button.click(button = "Remove All Items");
+
+			// To be removed when COMMERCE-10472 is fixed
+
+			Button.click(button = "Yes");
+
+			// To be removed when COMMERCE-10472 is fixed
+
+		}
+
+		task ("And selects 'MIN55860' on the quick add-to-cart search bar") {
+			CommerceQuickAddToCart.searchAndSelectOrderItem(orderItemSku = "MIN55860");
+		}
+
+		task ("And adds the SKU to the cart") {
+			Click(locator1 = "CommerceAccelerators#QUICK_MINI_CART_ADD_TO_CART_BUTTON");
+		}
+
+		task ("Then the SKU is added to the cart with quantity 3") {
+			AssertTextEquals(
+				key_cartItemSku = "MIN55860",
+				locator1 = "CommerceAccelerators#MINI_CART_ITEM_QUANTITY",
+				value1 = "3");
+		}
+
+		task ("And When the quantity is edited to '2'") {
+			CommerceNavigator.editMiniCartItemQuantity(
+				cartItemSku = "MIN55860",
+				productQuantity = "2");
+		}
+
+		task ("Then the quantity selector becomes red") {
+			KeyPress(
+				key_cartItemSku = "MIN55860",
+				locator1 = "CommerceAccelerators#MINI_CART_ITEM_QUANTITY",
+				value1 = "\ENTER");
+
+			AssertElementPresent(locator1 = "TextInput#ALERT");
+		}
+
+		task ("And When the quantity is edited to '4'") {
+			CommerceNavigator.editMiniCartItemQuantity(
+				cartItemSku = "MIN55860",
+				productQuantity = "4");
+		}
+
+		task ("Then the quantity is recognized as acceptable") {
+			KeyPress(
+				key_cartItemSku = "MIN55860",
+				locator1 = "CommerceAccelerators#MINI_CART_ITEM_QUANTITY",
+				value1 = "\ENTER");
+
+			AssertElementNotPresent(locator1 = "TextInput#ALERT");
+		}
+	}
+
+	@description = "This is a test for COMMERCE-10532. Buyer user can create first order adding SKU via quick add-to-cart"
+	@priority = "4"
+	test CanCreateFirstOrderAddingSKUViaQuickAdd {
+		property custom.properties = "jsonws.web.service.paths.excludes=${line.separator}company.security.strangers.verify=false";
+		property portal.acceptance = "false";
+
+		task ("Given an account with a buyer user is created") {
+			CommerceEntry.addAccountEntryUser(
+				accountName = "Commerce Account",
+				accountType = "Business",
+				agreeToTermsAndAnswerReminderQuery = "true",
+				createNewAccount = "true",
+				requireReset = "false",
+				userEmailAddress = "buyer@liferay.com",
+				userFirstName = "Buyer",
+				userLastName = "User",
+				userRole = "Buyer",
+				userScreenName = "buyeruser",
+				userSiteMembership = "Minium");
+		}
+
+		task ("And buyer user logs in") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+		}
+
+		task ("And the buyer user opens the minicart") {
+			FrontStore.addInitializerProductToCart(
```

Comment:
I guess also this will be removed, so please add another comment.

Reply:
I can see the comment already written right after the macro FrontStore.addInitializerProductToCart